app/lib/order_txt.rb

```diff
@@ -17,11 +19,39 @@ def to_txt
     text += '****** ' + I18n.t('orders.fax.to_address') + "\n\n"
     text += "#{FoodsoftConfig[:name]}\n#{contact[:street]}\n#{contact[:zip_code]} #{contact[:city]}\n\n"
     text += '****** ' + I18n.t('orders.fax.articles') + "\n\n"
-    text += format("%8s %8s   %s\n", I18n.t('orders.fax.number'), I18n.t('orders.fax.amount'),
-                   I18n.t('orders.fax.name'))
+
+    # prepare order_articles data
+    longest_number_string_length = 0
+    longest_amount_string_length = I18n.t('orders.fax.amount').length
+    longest_unit_string_length = I18n.t('orders.fax.unit').length
+    @order_positions = @order.order_articles.ordered.includes(:article_version).map do |oa|
```

Comment:
Here I'm unsure whether this needs to be an instance variable or if there are any adverse effects of this. I was a bit confused with the for loop in line 52, that's why I used an `@`